channels/ni.m3u

```diff
@@ -1,21 +1,25 @@
+#EXTINF:-1 tvg-id="Canal10.ni" tvg-country="NI" tvg-language="Spanish" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/4/4f/Canal_10_Nicaragua.png" group-title="Local",Canal 10 (576p) [Offline]
 https://cootv.cootel.com.ni/streams/d/SSLCH10-SD-ZUK/playlist.m3u8
-#EXTINF:-1 tvg-id="Canal11.ni" tvg-country="NI" tvg-language="Spanish" tvg-logo="https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcQm2ZYMFAUHTdq_BjIfNchqFkNhPo5eqjZm26zyahtBlWLjXDAZ" group-title="Local",Canal 11 (480p) [Offline]
+#EXTINF:-1 tvg-id="Canal11.ni" tvg-country="NI" tvg-language="Spanish" tvg-logo="https://static.wikia.nocookie.net/logopedia/images/b/b4/Canal_11_Nicaragua_2010_3.png" group-title="Local",TV RED Canal 11 (480p) [Offline]
 https://cootv.cootel.com.ni/streams/d/SSLCH11-SD-GZN/playlist.m3u8
-#EXTINF:-1 tvg-id="Canal12.ni" tvg-country="NI" tvg-language="Spanish" tvg-logo="https://vignette.wikia.nocookie.net/logopedia/images/6/6a/Nicavision1993.png/revision/latest?cb=20180426154048" group-title="Local",Canal 12 (720p) [Not 24/7]
+#EXTINF:-1 tvg-id="Canal12.ni" tvg-country="NI" tvg-language="Spanish" tvg-logo="https://vignette.wikia.nocookie.net/logopedia/images/6/6a/Nicavision1993.png/revision/latest?cb=20180426154048" group-title="Local",Nicavisión Canal 12 (720p) [Not 24/7]
 https://5ca9af4645e15.streamlock.net/canal12/videocanal12/playlist.m3u8
-#EXTINF:-1 tvg-id="MasTV.ni" tvg-country="NI" tvg-language="Spanish" tvg-logo="" group-title="",MasTV (Canal 23) [Offline]
-http://209.91.213.10:8088/play/a01p
-#EXTINF:-1 tvg-id="Canal8.ni" tvg-country="NI" tvg-language="Spanish" tvg-logo="" group-title="Local",tn8 (720p)
-https://60417ddeaf0d9.streamlock.net/tn8/videotn8/playlist.m3u8
-#EXTINF:-1 tvg-id="VivaNicaraguaCanal13.ni" tvg-country="NI" tvg-language="Spanish" tvg-logo="https://imagenes.gatotv.com/logos/canales/oscuros/13_de_nicaragua-mediano.png" group-title="Local",Viva Nicaragua Canal 13 (720p)
+#EXTINF:-1 tvg-id="VivaNicaraguaCanal13.ni" tvg-country="NI" tvg-language="Spanish" tvg-logo="https://static.wikia.nocookie.net/logopedia/images/9/97/VivaCanal13current.png" group-title="Local",Viva Nicaragua Canal 13 (720p)
 http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.dailymotion.com/vivanicaragua13
 #EXTINF:-1 tvg-id="wtv.ni" tvg-country="NI" tvg-language="Spanish" tvg-logo="https://wmediosnicaragua.com/img/LOGO-CANAL.png" group-title="Entertainment",WTV Canal 20 (720p)
 https://cloudvideo.servers10.com:8081/8130/index.m3u8
+#EXTINF:-1 tvg-id="vostv.ni" tvg-country="NI" tvg-language="Spanish" tvg-logo="https://static.wikia.nocookie.net/logopedia/images/8/86/VosTV_2010.png" group-title="local",Vos Tv
+http://ott.streann.com:8080/loadbalancer/services/public/channels/59e60c4997381ef50d15c041/playlist.m3u8
+#EXTINF:-1 tvg-id="MasTV.ni" tvg-country="NI" tvg-language="Spanish" tvg-logo="" group-title="",MasTV (Canal 23) [Offline]
```

Comment:
MasTV (Canal 23) is a Puerto-Rican channel, shouldn't be there and wi;ll be fixed in https://github.com/iptv-org/iptv/pull/5653